build/controllers/MimeTypeController.php

```diff
@@ -116,8 +112,13 @@ private function generateMimeTypesFile($outFile, $content)
  * https://svn.apache.org/viewvc/httpd/httpd/trunk/docs/conf/mime.types?view=markup
  * This file has been placed in the public domain for unlimited redistribution.
  */
-return $array;
+\$mimeTypes = $array;
+
+if (PHP_VERSION_ID >= 80100) {
```

Comment:
Why moving it here?

Reply:
@samdark At actual version this condition placed here https://github.com/yiisoft/yii2/blob/master/framework/helpers/mimeTypes.php#L1000 but i think it was added here manually. This condition in controller added without manual testing ) https://github.com/yiisoft/yii2/blob/master/build/controllers/MimeTypeController.php#L105 and not working. First parameter in `array_replace` must be array, but passing string.